CMSgov/mint-app
# [EASI-2556] Model Plan Created Email to MINT Dev Team

# EASI-2556

## Changes and Description

- Added MINT Dev Team Email to ENV VARS
- Added Dev Team Email to Email Service Config
- Added Model Plan Created Email to MINT Dev Team



## How to test this change

- Start the MINT dev environment
- Open MailCatcher at http://localhost:1085/
- Create a Model Plan
- Confirm that you received an email on MailCatcher to the MINT dev team

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
